Test/XLTest.cs

```diff
@@ -893,7 +893,7 @@ public static void XLSearchWithGeneratedIndices()
         {
             XLSearchTask xlSearchTask = new XLSearchTask
             {
-                CommonParameters = new CommonParameters()
+                CommonParameters = new CommonParameters(trimMsMsPeaks: false)
```

Comment:
I added peak filtering for .mgf files to the new mzLib version, which was missing. I edited this unit test to not filter MS2 peaks to keep the results of the test the same as the old version.